prepare_source

```diff
@@ -1,8 +1,20 @@
-apt_src containerd
+git_src --branch v2.0.4 https://github.com/containerd/containerd
 
-# patch 
-cp -r patches/remove-limitnofile.patch "$dir/src/debian/patches/"
-echo remove-limitnofile.patch >> "$dir/src/debian/patches/series"
+# debian specific things
+crd=$(mktemp -d)      
+git clone https://salsa.debian.org/go-team/packages/containerd.git $crd
+pushd $crd
+#git checkout debian/1.7.24_ds1-4
```

Comment:
assuming this comment is meant as documentation for the following line, i.e. that this commit is equivalent to the tag?

Reply:
generally speaking, the following steps are required:
1. get containerd upstream v2
2. get somewhat latest `debian/` folder (1.7.24_ds1-4)
3. make debian folder work with v2 ( <---thats the interesting part)

Later, if I have time I will check step 3  - how @5kt  you did it. But mostly looks good to me. 

@fwilhe you could try to build it locally and check the binaries, or try to understand why the changes are required to make it work with v2 - no must, just because you asked